crossfit/backend/torch/hf/memory_curve_utils.py

```diff
@@ -12,13 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from __future__ import annotations
 
 import joblib
 import numpy as np
 import torch
+import transformers
```

Comment:
Is it possible to skip this altogether?

Reply:
Discussed with Ayush, we import transformers across NeMo Curator and Crossfit so we can keep the import as is for now.